tests/grep_test.py

```diff
@@ -122,4 +122,4 @@ def test_grep_error(file_config_files, capfd, args):
     assert ret == 128
     out, err = capfd.readouterr()
     assert out == ''
-    assert err == 'fatal: no pattern given.\n'
+    assert 'fatal: no pattern given' in err
```

Comment:
the "only once" was an important test earlier I think you can change this test to be:

```python
assert err in {
    'fatal: no pattern given.\n',  # git < ...
    'fatal: no pattern given\n',  # git >= ...
}
```

Reply:
Thanks for the suggestion